feat: Document KeymanWeb API calls returning a promise (#449)
keymanapp/help.keyman.com: developer/engine/web/15.0/reference/core/init.md

  * If `'manual'`, KeymanWeb must be instructed to attached manually to each
    control it should handle input for.

`setActiveOnRegister`

: `string` <span class="optional">optional</span>

  Specifies whether KeymanWeb will set the newly registered keyboard as active. Default value `true`.

  * If `true`, KeymanWeb will automatically activate a keyboard when registered.
  * If `false`, KeymanWeb will not activate a keyboard when registered.

`useAlerts`

: `boolean` <span class="optional">optional</span>

>   Specifies whether KeymanWeb will display internal alerts. Default value `true`.

Reviewer: ```suggestion
  Specifies whether KeymanWeb's alert feedback should be enabled. Default value `true`.
```

>   * If `true`, KeymanWeb will display internal alerts.

Reviewer: ```suggestion
  * If `true`, KeymanWeb will display its default alerts.
```

>   * If `false`, KeymanWeb will not display internal alerts. Choose this if you want to customize the alert messages for use on your site.

Reviewer: ```suggestion
  * If `false`, KeymanWeb will not display its default alerts. Choose this if you wish to disable them or would prefer to customize your site's error feedback.
```

> 

Reviewer: Also:

```suggestion
  * In either case, any calls your page makes to `keyman.util.alert()` will not be blocked.  This only affects calls built into KeymanWeb itself.

```
